subprojects/core-api/src/main/java/org/gradle/api/internal/plugins/software/SoftwareType.java

```diff
@@ -35,14 +35,14 @@
     /**
      * The name of the software type.  This will correspond to the DSL element that is exposed to configure the software type.
      *
-     * @since 8.8
+     * @since 8.9
      */
     String name();
 
     /**
      * The model type used to configure the software type.  Note that the actual return type of the method may be a subtype of this type.
      *
-     * @since 8.8
+     * @since 8.9
```

Comment:
I was actually trying to communicate that the method this annotates should return a type that is the same as or a subclass of what this method returns.  I've reworded this to be less vague.  Let me know if you think this could still be better.